client/my-sites/backup/multisite-no-backup-plan-switch.tsx

```diff
@@ -0,0 +1,67 @@
+import { WPCOM_FEATURES_BACKUPS } from '@automattic/calypso-products';
+import { FunctionComponent, ReactNode, useCallback } from 'react';
+import { useSelector } from 'react-redux';
+import QuerySiteFeatures from 'calypso/components/data/query-site-features';
+import RenderSwitch from 'calypso/components/jetpack/render-switch';
+import getFeaturesBySiteId from 'calypso/state/selectors/get-site-features';
+import isRequestingSiteFeatures from 'calypso/state/selectors/is-requesting-site-features';
+import siteHasFeature from 'calypso/state/selectors/site-has-feature';
+import { isJetpackSiteMultiSite } from 'calypso/state/sites/selectors';
+import getSelectedSiteId from 'calypso/state/ui/selectors/get-selected-site-id';
+
+type Props = {
+	trueComponent: ReactNode;
+	falseComponent: ReactNode;
+	loadingComponent?: ReactNode;
+};
+
+const MultisiteNoBackupPlanSwitch: FunctionComponent< Props > = ( {
+	trueComponent,
+	falseComponent,
+	loadingComponent,
+} ) => {
+	const siteId = useSelector( getSelectedSiteId );
+
+	const isMultiSite =
+		useSelector( ( state ) => siteId && isJetpackSiteMultiSite( state, siteId ) ) || false;
```

Comment:
```suggestion
		useSelector( ( state ) => siteId && isJetpackSiteMultiSite( state, siteId ) );
```

A left-over from debugging?

Reply:
@macbre The `isJetpackSiteMultiSite` can also be `null`, but for this use case it would be better to always return a boolean, so in case it is null I'm returning false, so we could use it for the `renderCondition` prop in `RenderSwitch`. Maybe there's a better way to handle it in TS? Thoughts?

Reply:
Oh, then this code does make sense.